docs: add SEO frontmatter, llms.txt, and code-block hygiene (#2394)
php/frankenphp: docs/compile.md

---

> title: Compile FrankenPHP From Sources With PHP as a Library

Reviewer: ```suggestion
title: Compile FrankenPHP from sources with libphp.so
```

> description: Build FrankenPHP from source on Linux, macOS and FreeBSD, link PHP as a dynamic library via xcaddy or go build, and add custom Caddy modules and extensions.

Reviewer: ```suggestion
description: Build FrankenPHP from source on Linux, macOS and FreeBSD, linking PHP as a shared library via xcaddy or go build, and add custom Caddy modules and extensions.
```
